elements/lisk-codec/src/bytes.ts

```diff
@@ -11,14 +11,12 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
-import { writeVarInt } from './varint';
+import { writeUInt32, readUInt32 } from './varint';
 
-interface SchemaProperty {
-	readonly dataType: string;
-}
-
-export const writeBytes = (bytes: Buffer, _schema: SchemaProperty): Buffer =>
-	Buffer.concat([writeVarInt(bytes.length, { dataType: 'uint32' }), bytes]);
+export const writeBytes = (bytes: Buffer): Buffer =>
+	Buffer.concat([writeUInt32(bytes.length), bytes]);
```

Comment:
this function should hint at it being a varint without having to look at the import guess it's ok in this small file